utils/plot-tasks.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
```

Comment:
Please rename to plot_tasks, and remove this pylint override:
https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming 
https://google.github.io/styleguide/pyguide.html?showone=Main#Main
"Even a file meant to be used as a script should be importable"

Yes, we have other files that aren't compliant at present, but we should fix those up.

Reply:
done